client/jetpack-cloud/sections/onboarding-tours/add-new-site-tour-step-1/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { useTranslate } from 'i18n-calypso';
+import GuidedTour from 'calypso/jetpack-cloud/components/guided-tour';
+import { JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME } from '../constants';
+
+export default function AddNewSiteTourStep1() {
+	const translate = useTranslate();
+	const urlParams = new URLSearchParams( window.location.search );
+	const shouldRenderAddSiteTourStep1 = urlParams.get( 'tour' ) === 'add-new-site';
+
+	return (
+		shouldRenderAddSiteTourStep1 && (
+			<GuidedTour
+				preferenceName={ JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME[ 'addSiteStep1' ] }
+				tours={ [
+					{
+						target: '#sites-overview-add-sites-button .split-button__toggle',
+						popoverPosition: 'bottom left',
+						title: translate( 'Click the arrow button' ),
+						description: (
+							<>
+								{ translate( 'Click the arrow button and select "Connect a site to Jetpack".' ) }
+								<br />
+								<br />
+								{ translate(
+									'Sites with jetpack installed will automatically appear in the site management view.'
+								) }
+							</>
+						),
+
+						nextStepOnTargetClick: '#sites-overview-add-sites-button .split-button__toggle',
```

Comment:
If we want to target the entire component (the Add new site button and the dropdown arrow), we should change `'#sites-overview-add-sites-button .split-button__toggle'` to `'#sites-overview-add-sites-button .split-button'`.